R/cor_test_biserial.R

```diff
@@ -61,8 +61,8 @@
   #   x = var_x,
   #   y = var_y
   # )[1])
-  m1 <- mean(var_x[var_y == 1])
-  m0 <- mean(var_x[var_y == 0])
+  m1 <- mean(var_x[var_y == unique(var_y)[1]])
+  m0 <- mean(var_x[var_y == unique(var_y)[2]])
   sn <- stats::sd(var_x)
   q <- mean(var_y)
```

Comment:
`q <- mean(var_y)` computes the proportion, which only works when the values are 0/1...